.github/workflows/convertKqlFuncYamlToArmTemplate.yaml

```diff
@@ -0,0 +1,62 @@
+name: Convert Kql function yaml to ARM template
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize, closed, labeled]
+    paths:
+       - 'Parsers/ASimDns/Parsers/**'
+       - 'Parsers/ASimNetworkSession/Parsers/**'
+       - 'Parsers/ASimWebSession/Parsers/**'
+jobs:
+  kqlFuncYaml2Arm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Doesn't the first set require a name?

Reply:
Name property is not optional, but it is good for documentation and logs.
Adding name